keras_nlp/layers/transformer_decoder.py

```diff
@@ -114,11 +127,13 @@ def _build(self, input_shape):
         # First dense layer in the feedforward network, which maps input
         # feauture size to dimension `self.intermediate_dim`.
         self._intermediate_dense = keras.layers.Dense(
-            self.intermediate_dim, activation=self.activation
+            self.intermediate_dim, activation=self.activation, 
```

Comment:
Did you run black on this? I'm pretty sure the formatter would push this to an argument a line.

Reply:
Hey
Thanks for the review, I didn't run black, I'll run it on these!